app/javascript/helpers/ajaxRequestHelper.js

```diff
@@ -0,0 +1,73 @@
+import {XMLHttpRequest} from 'xmlhttprequest';
+
+export class AjaxRequest {
```

Comment:
As this class doing network requests
![Screen Shot 2021-07-12 at 1 49 46 PM](https://user-images.githubusercontent.com/6076762/125243255-0b1b7080-e318-11eb-8b2d-c82efbfa59ff.png)
, it should be placed under `adapters` folder instead of `helpers`

Reply:
Assuming I would have kept this file, after re-thinking, yes it seems more appropriate as an adapter.
I got a bit confused by the `we usually create one file per API resource`, as this AjaxRequest object is more general and does not have any dependency with a specific resource. So it appeared more generic = more a helper at first hand. 
But it totally matches the `async tasks` part of an adapter, so yeah, seems ok! ;-) 

@junan i'm still resolving this conversation as this file was not supposed to be in here.